### ISSUE TITLE
Add Page: Full Wallet Example

### ISSUE THREAD

**Issue by riverKanies** [bitcoindevkit/book-of-bdk#74]
**TLDR** seems we have two Options:
- command line wallet app with all 4 functions in a single file/example
- 4 separate example folders, and the full wallet page just links to them and describes how to use them in order

**Notes**:
Thinking more about the full wallet example, I realized we can’t have seed generation and tx broadcasting in the same example because we need to get sats (from mutiny for now). So instead we can break the wallet examples into parts:

1. seed gen and descriptor creation (basically the current descriptor example)

2. wallet recovery and balance display (same as quickstart but with custom seeds to gen descriptors, also an example of this in descriptor section). also persistence (user input seeds)

3. load from persistence and display balance (run this to confirm sats received from mutiny net)

4. load from persistence and build and send transaction (user input sats)

5. rerun load and display to see sats reduced

so should we walk the reader through building a command line wallet app with these 4 commands (gen, rec, load, send)? note that gen and rec functionality is currently outlined in the ‘keys and descriptors’ section of the book. Perhaps these should all be separate example projects and the full-wallet page just outlines the order in which one should navigate them (with links to them ofcourse).

what do you think @thunderbiscuit ?

**Comment by riverKanies** [comment]
moving forward with isolated examples for now. full-wallet page will just be a walkthrough of 4 examples linearly for this first version of it

**Comment by thunderbiscuit** [comment]
I think this is now ready to be closed, thanks for the work @riverKanies!
